web/declarations.d.ts

```diff
@@ -0,0 +1,14 @@
+
+
+declare module '*.woff2' {
+  const content: string;
+  export default content;
+}
+
+
+declare module '!url-loader*' {
+  const content: string;
+  export default content;
+}
+
+declare module 'vue-resize-directive';
```

Comment:
What's going on in this file?

Reply:
there are no typing information for libraries like `vue-resize-directive`. `.d.ts` files are definition files to tell TypeScript what the types of certain libraries are.